intelmq/bots/collectors/mail/collector_mail_url.py

```diff
@@ -38,9 +38,12 @@ def process(self):
                 self.logger.info("Reading email report")
 
                 for body in message.body['plain']:
-                    match = re.search(self.parameters.url_regex, body)
+                    match = re.search(self.parameters.url_regex, str(body))
```

Comment:
On Tue, May 24, 2016 at 02:24:19AM -0700, Sebastian wrote:
> Why should this not be a string? Could this be a bytes in a special case? Could you describe/show such a case?

All I can say right now is that I observed this behavior while
testing the bot with a real-world message.  This is the traceback:

  2016-05-27 12:09:13,119 - cymru - ERROR - Bot has found a problem.
  Traceback (most recent call last):
    File "/usr/local/lib/python3.4/dist-packages/intelmq-1.0.0.dev4-py3.4.egg/intelmq/lib/bot.py", line 102, in start
      self.process()
    File "/usr/local/lib/python3.4/dist-packages/intelmq-1.0.0.dev4-py3.4.egg/intelmq/bots/collectors/mail/collector_mail_url.py", line 41, in process
      match = re.search(self.parameters.url_regex, body)
    File "/usr/lib/python3.4/re.py", line 170, in search
      return _compile(pattern, flags).search(string)
  TypeError: can't use a string pattern on a bytes-like object